README.md

```diff
@@ -35,7 +36,23 @@ Convert the Ruby and ActiveRecord code below into Javascript and Mongoose code:
 ```
 
 ```js
-// Your answer...
+
+show: function(req){
+  Instructor.findOne({"name": req.name}, function(err, docs){
+    console.log(docs);
+    return docs;
+  })
+}
+
+instructorsController.show({name: "Andy"});
+
+WishlistItems.create({description: "Resin Laying Deer Figurine, Gold"}, (err, wishlistitems) => {
```

Comment:
`WishlistItems.create({description: "Resin Laying Deer Figurine, Gold"})`

This is looks like Ruby syntax here